build/bin/install_tarball

```diff
@@ -2,7 +2,8 @@
 
 set -eu
 
-PMM_DIR="$PWD/usr/local/percona/pmm2"
+PMM_DIR=${PMM_DIR:-/usr/local/percona/pmm2}
```

Comment:
Default path will require root. It is the same path as used by pmm-agent to find exporters by default.
Both can be changed by user.